mangum/adapter.py

```diff
@@ -42,6 +42,8 @@ def get_logger(log_level: str) -> logging.Logger:
     return logger
 
 
+D = typing.TypeVar('D')
```

Comment:
Let's remove this. I think it's fine to sprinkle `# type: ignore` where needed, and then revise the types later in a way that doesn't hurt readability.